packages/valaxy/node/cli/new.ts

```diff
@@ -0,0 +1,39 @@
+import type { Argv } from 'yargs'
+
+export const newPost = (cli: Argv<{}>) => {
+  cli.command(
+    'new <title>',
+    'Draft a new post',
+    (args) => {
+      args.usage('$0 <title> -p [path] -l [layout]')
+        .positional('title', {
+          describe: 'The title of the new post',
+          required: true,
+        })
+        .option('path', {
+          alias: 'p',
+          type: 'string',
+          describe: 'the path to generate new post',
+          default: 'post',
+        })
+        .option('slug', {
+          alias: 's',
+          type: 'string',
+          describe: 'slug',
+        })
+        .option('layout', {
+          alias: 'l',
+          type: 'string',
+          default: 'post',
+        })
+        .option('date', {
+          alias: 'd',
+          type: 'boolean',
+          describe: 'Generate post with the current date',
+        })
+        .strict()
+        .help()
+    }, async (_args) => {
+      // TODO: Implement commands
```

Comment:
Will it be finished in this PR?

Reply:
No, the implementation will be included in another PR.